tests/ts/examples.ts

```diff
@@ -12,6 +13,27 @@ const { lit, raw, ref } = objection;
 
 class CustomValidationError extends Error {}
 
+class CustomValidator extends objection.Validator {
```

Comment:
I think we might want to add a quick comment here saying this is not example code for people to read to understand the API, but just the minimal stubs to exercise the typings, perhaps?

Reply:
Done.